wheretofindme/models.py

```diff
@@ -96,15 +98,29 @@ class UserManager(BaseUserManager.from_queryset(UserQuerySet)):
 class User(AbstractUser):
     objects = UserManager()
 
+    search_enabled = models.BooleanField(default=False)
+
     def follows(self):
         return [f.to_user for f in self.follow_set.prefetch_related("to_user")]
 
     def first_three(self):
         return self.internetidentity_set.exclude(name="")[:3]
 
+    def first_three_ellipsis(self):
```

Comment:
It might be worth folding this into `first_three`:

```python
def first_three(self):
  records = self.internetidentity_set.exclude(name="")[:4]
  return records[:3], len(records) > 3
```

Doing a full count of the user's aliases is likely to do a lot more IO than needed to determine that they have more than three, which may be an issue as the service grows (and will definitely be an issue if someone logs thousands of identities on a single user).